src/number_theory/legendre_symbol/quadratic_char.lean

```diff
@@ -58,6 +58,16 @@ begin
   exact char_p.neg_one_ne_one _ (ring_char F),
 end
 
+/-- Characteristic `≠ 2` implies that `-a ≠ a` when `a ≠ 0`. -/
+lemma neg_ne_self_of_char_ne_two (hF : ring_char F ≠ 2) {a : F} (ha : a ≠ 0) : a ≠ -a :=
```

Comment:
Could you please move this to the file on `ring_char`?

Reply:
The assumption `ring_char F ≠ 2` (which is what I need here) is only sufficient when the characteristic cannot be an even number > 2, so the statement is not true for general rings. Also, it is just a variation of the preceding lemma, so it is not clear to me why this should be moved and the other one not.
Since I may need to add some further lemmas on finite fields later on, I would suggest to leave all of this here for now, and when I'm essentially done with working on Legendre symbols, we can discuss where it makes sense to put the lemmas in this section (perhaps somewhere under `field_theory.finite`).

Reply:
Yeah, I think a section should be added to `field_theory.finite`.

Reply:
Would it be OK to first collect the relevant lemmas here and eventually move them all together?
Or is the way to go to put them at the "right" place immediately?

Reply:
In this case, let's leave them here for now. But in the near future, it would be good to move them to the "right" place.